packages/office-addin-dev-settings/package.json

```diff
@@ -22,6 +22,7 @@
     "Office Add-in"
   ],
   "dependencies": {
+    "@microsoft/teamsfx-cli": "0.11.0-rc.0",
```

Comment:
I am not aware of what this `-rc.0` mean. Can't we use the normal `^0.11` version?

Reply:
Presumably "rc" stands for "release candidate".  This is basically putting an unofficial version out there for us to try out before they make it part of an official release.  We'll need to update this once the official release includes the command we need.